this_is_a_test.py

```diff
@@ -1,2 +1,7 @@
 # My passwords
 a_password = "my_sp3c1f1c_pa$$w0rd_trav1s_ll0yd_1"
+this_is_a_password = "nbv5nyc@cdq2pjw9JPZ"
+test_val = 'test'   
+# another comment
+another_secret = 'pnh4ZBA*wen-zpb0vby'
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 03bd8fb1-581f-4626-9396-62fc1146bde4 -->